app/components/pages/CustomFieldsSettings/tests/CreateEditCustomField.test.js

```
@@ -0,0 +1,160 @@
+import '@testing-library/jest-dom';
+import { injectIntl } from 'react-intl';
+import userEvent from '@testing-library/user-event';
+import { screen, render } from '../../../../utils/test-utils';
+
+import { CreateEditCustomField, mapDispatchToProps } from '../CreateEditCustomField';
+import { CustomFieldsSettingsMode } from '../constant';
+import globalMessages from '../../Cap/messages';
+import { mockCustomFieldMeta } from './mockData';
+
+export const setModeMock = jest.fn();
+export const setFieldNameMockFn = jest.fn();
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)